contrib/test_jenkins.sh

```diff
@@ -1265,7 +1265,8 @@ test_memtrack() {
 test_unused_env_var() {
 	# We must create a UCP worker to get the warning about unused variables
 	echo "==== Running ucx_info env vars test ===="
-	UCX_IB_PORTS=mlx5_0:1 ./src/tools/info/ucx_info -epw -u t | grep "unused" | grep -q "UCX_IB_PORTS"
+	#UCX_TCP_CM_ALLOW_ADDR_INUSE is currently set but not used. Remove when tcp_sockcm is set in the cms priority list
+	UCX_IB_PORTS=mlx5_0:1 ./src/tools/info/ucx_info -epw -u t | grep "unused" | grep -q -E "UCX_IB_PORTS|UCX_TCP_CM_ALLOW_ADDR_INUSE"
```

Comment:
this line expects to find UCX_IB_PORTS in the output as unused env var to adding UCX_TCP_CM_ALLOW_ADDR_INUSE here is incorrect logic
can we set tcp_sockcm in the priority list for this ucx_info?

Reply:
Why incorrect? It's expecting either one of them and once the cms fallback is merged, this (or another) w/a will be removed..

Reply:
this line passes if UCX_IB_PORTS if reported, and fails if UCX_IB_PORTS is not reported
why do we need to add UCX_TCP_CM_ALLOW_ADDR_INUSE here? IMO it should pass even without adding it

Reply:
But because this new env param is set but not used, it does what this test checks.

Reply:
Will the test fail if we don't add it?
If not, why need to bother add it just to remove later?

Reply:
Yes, i added it because jenkins failed.
This new env param appears in the output of grep here (since there is no cms fallback yet and tcp isn't in the priority list).

Reply:
i'm probably missing something but why would the test fail if the new parameter appeared in the grep? didn't the old parameter appear as well?

Reply:
also, doesn't it cause a warning in gtest?

Reply:
If the new param is set then UCX_IB_PORTS doesn't appear, only the new param, so it failed because it didn't find UCX_IB_PORTS.

Which warning in gtest do you mean? This one?
Without the cms fallback PR, tcp isn't set in the priority list and so gtest of ucp doesn't test tcp_sockcm

Reply:
ok